### ISSUE TITLE
argument "-rollup_top_k" does not work

### ISSUE THREAD

**Issue by dpajin-kentik** [kentik/ktranslate#12]
In ktranslate v2 I was not able to get Top K results using argument `-rollup_top_k <value>`. I always get all the results, I have tried with topk values 5, 10 and 15.

**Comment by i3149** [comment]
Can you try now with the latest image. Should be working.

**Comment by dpajin-kentik** [comment]
Almost there... When I use one rollup, the number of returned outputs is for 1 higher than configured with topk. So if I configure 10, I get 11 results, for 15 I got 16, for 20 I got 21. 

The other thing I tried with two rollups and topk 5 -  for the first rollup I get 6 results, but for the second I get sometimes 6, sometimes 7 and even 8 results.

When I tried with 3 rollups, first two results are always 6 and the last one varies 6-8. 

I also noticed that the output file for JSON format is not valid JSON. It just glues JSON results together -  I will make a separate issue for this.
